chalicelib/lib/slack.py

```diff
@@ -147,14 +147,21 @@ def slack_payload_extractor(req):
         return "failed extracting payload", 200
 
 
-def verify_token(headers, body, token):
+def verify_token(headers, body, signing_secret):
+    """
+    https://api.slack.com/docs/verifying-requests-from-slack
+
+    1. Grab timestamp and slack signature from headers.
+    2. Concat and create a signature with timestamp + body
+    3. Hash the signature together with your signing_secret token from slack settings
+    4. Compare digest to slack signature from header
+    """
     request_timestamp = headers['X-Slack-Request-Timestamp']
-    # create basestring out of v0 + timestamp + body
```

Comment:
No need to have this comment since your code explains itself 😄

Reply:
ok will remove